modelarmor/src/main/java/modelarmor/CreateTemplate.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package modelarmor;
+
+import com.google.cloud.modelarmor.v1.CreateTemplateRequest;
+import com.google.cloud.modelarmor.v1.DetectionConfidenceLevel;
+import com.google.cloud.modelarmor.v1.FilterConfig;
+import com.google.cloud.modelarmor.v1.LocationName;
+import com.google.cloud.modelarmor.v1.ModelArmorClient;
+import com.google.cloud.modelarmor.v1.ModelArmorSettings;
+import com.google.cloud.modelarmor.v1.RaiFilterSettings;
+import com.google.cloud.modelarmor.v1.RaiFilterSettings.RaiFilter;
+import com.google.cloud.modelarmor.v1.RaiFilterType;
+import com.google.cloud.modelarmor.v1.Template;
+import com.google.protobuf.util.JsonFormat;
+import java.util.List;
+
+public class CreateTemplate {
+
+  public static void main(String[] args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String locationId = "your-location-id";
+    String templateId = "your-template-id";
+
+    createTemplate(projectId, locationId, templateId);
+  }
+
+  public static Template createTemplate(String projectId, String locationId, String templateId)
+      throws Exception {
```

Comment:
issue: be more explicit about the type of exception thrown -- I think it's `IOException`.

Reply:
Here and below.